pkg/catalogv2/system/system.go

```diff
@@ -124,7 +124,7 @@ func (m *Manager) Start(ctx context.Context) {
 	m.clusterRepos.OnChange(ctx, "catalog-refresh-trigger", m.onTrigger)
 }
 
-func (m *Manager) onSetting(key string, obj *v3.Setting) (*v3.Setting, error) {
+func (m *Manager) onSetting(_ context.Context, key string, obj *v3.Setting) (*v3.Setting, error) {
```

Comment:
Manager already has `ctx context.Context` in its struct; there are probably a lot of places like this where context has been built into the controller because we didn't have a way to plumb it through function calls. That will all need to be redone.